src/api/java/com/minecolonies/api/items/ItemBlockHut.java

```diff
@@ -47,8 +47,10 @@ public void addInformation(@NotNull ItemStack stack, World worldIn, @NotNull Lis
         super.addInformation(stack, worldIn, tooltip, flagIn);
         if (block.needsResearch())
         {
-            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_1, block.getName()));
-            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_2, block.getName()));
+            final TranslationTextComponent blockName = new TranslationTextComponent(block.getTranslationKey());
+            //TranslationTextComponent blockName = new TranslationTextComponent(block.getName());
+            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_1, blockName));
+            tooltip.add(new TranslationTextComponent(TranslationConstants.HUT_NEEDS_RESEARCH_TOOLTIP_2, blockName));
```

Comment:
Since this only fires on the client side, you can also use block.getTranslatedName(), which saved a little bit of verbosity. It's the approach I took https://github.com/ldtteam/minecolonies/blob/6a51ce593e80dbf1ea378746dc596cb2dd00124f/src/api/java/com/minecolonies/api/items/ItemBlockHut.java and seems to work fine.